api/src/functionalTest/java/uk/gov/hmcts/payment/functional/dsl/PaymentsTestDsl.java

```diff
@@ -61,6 +61,13 @@ public PaymentGivenDsl returnUrl(String url) {
             return this;
         }
 
+        public PaymentGivenDsl setFF4Jfeature(String featureName, boolean value) {
+            String path = "/api/ff4j/store/features/" + featureName + (value ? "enable" : "disable");
```

Comment:
atm ff4j flags are stored in memory.. so there is no guarantee  which instance is updated (aat has 2 instances). So, the tests become flakey until we address flags persistency.

Reply:
ok, for now I'm not using this functionality anywhere in the functional tests, but might come in handy later